packages/bundle-size/src/commands/uploadReport.js

```diff
@@ -0,0 +1,139 @@
+const { odata, TableClient, TablesSharedKeyCredential, TableTransaction } = require('@azure/data-tables');
+const chalk = require('chalk');
+
+const collectLocalReport = require('../utils/collectLocalReport');
+const { hrToSeconds } = require('../utils/helpers');
+
+/** @typedef {import('../utils/collectLocalReport').BundleSizeReportEntry} BundleSizeReportEntry */
+
+const AZURE_STORAGE_ACCOUNT = 'fluentbundlesize';
+const AZURE_STORAGE_TABLE_NAME = 'latest';
+const AZURE_ACCOUNT_KEY = process.env.BUNDLESIZE_ACCOUNT_KEY;
+
+/**
+ *  @param {BundleSizeReportEntry} entry
+ *
+ * @return {string}
+ */
+function createRowKey(entry) {
+  // Azure does not support slashes in "rowKey"
+  // https://docs.microsoft.com/archive/blogs/jmstall/azure-storage-naming-rules
+  return `${entry.packageName}${entry.path.replace(/.fixture.js$/, '').replace(/\//g, '')}`;
```

Comment:
shouldn't we properly escape those dots in regex?

Reply:
Good catch, thanks 👍 Applied in 9f3e270.